src/uris/deepLinks/deepLinkService.ts

```diff
@@ -1386,6 +1385,34 @@ export class DeepLinkService implements Disposable {
 						repository: repo,
 						source: 'launchpad',
 					} satisfies ShowWipArgs);
+					action = DeepLinkServiceAction.OpenAllPrChanges;
```

Comment:
Right now this would work because we really only open inspect on PR switches, in which case all the info is there to show the PR changes. Would be a nice additional touch though to check the action type in `this._context.action` though and maybe restrict/guard against any non-PR case.